joomla/joomla-cms
# [4.0] Update npm dependencies

Pull Request for Issue # .

### Summary of Changes
- Major versions update for few  dependencies
- Fixing the commander B/C breaks
- Removed files from diff and mediaelement as the dist folder doesn't provide them

### Testing Instructions
- Run `npm install` before applying this PR and make a copy of the `media` folder
- Apply this PR, run `npm install` and compare the `media` with the one you stored in the previous step. They should be identical (apart from the missing `diff.min.js` and `css/mejs-controls.png` which were removed upstream)

- Check that the media manager works correctly: Double-clicking on a folder navigates to that folder

Some changes regarding the `new Regex()`:
- Check that editing the colour in the Atum template works as expected
- Check that custom fields change context when selecting a different category
- Check that installing extensions from the JED is working correctly
- Check that Editing a user by changing the username and te password works as expected

- Also check that all all npm commands still work
- And finally confirm that CSS/JS is not broken, front end/ back end

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request



### Documentation Changes Required

No. @wilsonge @bembelimen after 4.0.5 could you have some people testing this one? Most dependencies are way behind...
Also please consider adding either dependabot or renovate bot so PRs are automatically created on each new update of any dependency
